static/css/starter-template.css

```diff
@@ -29,6 +29,7 @@ video {
 .hide-snippets {
   display: none !important;
 }
+
```

Comment:
Let's avoid whitespace changes unless this is deliberate?

Reply:
done